developer/src/kmc-ldml/test/helpers/index.ts

```diff
@@ -172,6 +174,16 @@ export function checkMessages() {
   assert.isEmpty(compilerTestCallbacks.messages, compilerEventFormat(compilerTestCallbacks.messages));
 }
 
+/** These tests aren't prepared for line number information in messages. Remove it so that comparisons pass. */
+function zapMessageMetadata() {
```

Comment:
ahhh - yes, zap and scrub have the same function. That's what I get for a long chain of PRs.

In a future PR, I remove `zapMessageMetadata()` and just use scrubContextFromMessages() instead.

Reply:
first commit on https://github.com/keymanapp/keyman/pull/14100